packages/components/src/styles/components/dialog-primitive.scss

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// DIALOG PRIMITIVE
+//
+
+$hds-dialog-primitive-horizontal-padding: 24px;
```

Comment:
This is a nit pick (and also still trying to understand naming conventions here), but since we use 24px for both horizontal and vertical (like the body padding) shouldn't we use a naming convention that reflects maybe something more agnostic (like maybe md and lg instead of horizontal/vertical-padding)?

Reply:
I'm with you but I don't think it's going to happen at least not anytime soon...

Reply:
@majedelass there's been a long ongoing discussion about spacing tokens. The answer is still no at the moment: no tokenization of spacing values. I'll send you in DM all the links for context.